blottertrax/description_provider.py

```diff
@@ -80,7 +80,7 @@ def format_network_to_friendly_name(info) -> str:
         social_network = ['twitter.com', 'facebook.com', 'instagram.com']
         link_type = info['type']
         target = info['target']
-        target = target.replace('(', '\(').replace(')', '\)')
+        target = target.replace('(', r'\(').replace(')', r'\)')
```

Comment:
Not sure these are functionally identical?  The raw string now means we should drop the slash for the escaping, yea?

Reply:
Hmm - The aim here is to add an extra `\` (presumably so it can be escaped further down the line). In either case no characters are escaped, but the r-string avoids any ambiguity.

Simple enough to test:
```
print('\(')
print(r'\(')
```
output the same.

Reply:
Oh you're definitely right. This is used to output the list of sites in proper markdown later on.

Going to chalk my comment up to being tired! 😅